doc/source/whatsnew/v0.20.0.txt

```diff
@@ -194,6 +195,31 @@ Using ``.iloc``. Here we will get the location of the 'A' column, then use *posi
   df.iloc[[0, 2], df.columns.get_loc('A')]
 
 
+.. _whatsnew_0200.api_breaking.deprecate_plotting
+
+Deprecate .plotting
+^^^^^^^^^^^^^^^^^^^
+
+``pandas.tools.plotting`` module has been deprecated, moving directory under the
```

Comment:
The ``pandas.tools.plotting`` module has been deprecated, in favor of the top level namespace ``pandas.plotting``